ketcher-autotests/tests/Indigo-Tools/Layout/layout.spec.ts

```diff
@@ -27,7 +28,7 @@ test.describe('Indigo Tools - Layout', () => {
 
   test('Center molecule after layout', async ({ page }) => {
     // Related Github issue: https://github.com/epam/ketcher/issues/2078
-    const anyStructure = 'benzene-rings.mol';
+    const anyStructure = 'Molfiles-V2000/benzene-rings.mol';
     await openFileWithShift(anyStructure, page);
```

Comment:
Why is this function used when opening a file? What does this function do?

Reply:
Actually I think instead of this function it could be openFileAndAddToCanvas() method